src/QuickMarcEditor/QuickMarcEditorRows/FixedField/FixedFieldFactory.js

```diff
@@ -49,13 +49,21 @@ export const FixedFieldFactory = {
         item.isArray = false;
       }
 
-      const value = content[item.code] || '';
+      const getValue = () => {
+        if (content[item.code]) {
+          return content[item.code];
+        }
+
+        return item.isArray
+          ? new Array(item.length).fill('\\')
+          : new Array(item.length + 1).join('\\');
```

Comment:
Could you add some comments why we need length +1?

Reply:
The reasone for this is that 
new Array(item.length).join('\\') dosn't work without fill like new Array((item.length).fill('\\').join('') 
I used equivalent without fill new Array(item.length + 1).join('\\');

As U wish It could be change to more readable to new Array((item.length).fill('\\').join('')

Reply:
Yes, let's use ` new Array((item.length).fill('\').join('')` it'll be easier to understand